### ISSUE TITLE
[20201015-pre Testing] Timelines added manually using psort are not available in the Timesketch UI

### ISSUE THREAD

**Issue by cshanahan** [google/timesketch#1417]
**TS Version**
Timesketch 20201015-pre installed manually using pip3

**Plaso Version**
Plaso-20201007 from GIFT PPA

**OS Version**
Ubuntu 18.04.5

**STEPS TO REPRODUCE**
- Use psort from Plaso to ingest a plaso DB file into Elasticsearch
  - Example command: psort.py --output_time_zone UTC -o timesketch --name <name_in_timesketch> --index <my_elastic_index_name> plaso_file.plaso
- Run `tsctl run_server` log in to the app and create a new Sketch
- From the Sketch UI the new index is not available as an option to add to the Sketch (it doesn't appear in the UI)

The index in Elasticsearch is fine. I can explore the index using other apps, including Kibana. This process works well and without error using Plaso v20200717 and Timesketch v20200710-pre.

If it helps, Elasticsearch is pinned to v7.5.1

Happy to help test.

**Comment by kiddinn** [comment]
Is the index available in the backend?

That is can you see the index available in the API client?

```
from timesketch_api_client import config

ts = config.get_client()
[(x.name, x.index_name, x.status) for x in ts.list_searchindices()]
```

**Comment by cshanahan** [comment]
Correct, the backend does include both timelines (I added two timelines as a test) using the API client .

**Comment by cshanahan** [comment]
I repeated the installation process again using a new Ubuntu 18.04.5 VM. I followed the same steps as before, with one exception; I changed the requirements.txt file to account for a newer version of celery (5.0.0) and it's dependencies (i.e. WTForms, and markdown) before I tried to install Timesketch.

In my first test, I allowed pip3 to complete the installation then manually upgraded celery and it's dependencies.

Now I am able to access timelines from the Timesketch UI as expected.

I don't know if the changes to packages would account for this behavior, or if the first failed/fixed installation process caused the issue. Point is, the issue is resolved and Timesketch v20201015-pre installs and works as expected using pip3 from source (not from PyPi).

If you don't have any questions from this I will close this as unfounded ?
